cfme/physical/provider/__init__.py

```diff
@@ -14,14 +14,14 @@
 
 class PhysicalProvider(Pretty, BaseProvider, Fillable):
     """
-    Abstract model of an infrastructure provider in cfme. See VMwareProvider or RHEVMProvider.
+    Abstract model of a physical infrastructure provider in cfme. See LenovoProvider.
     """
     provider_types = {}
     category = "physical"
     pretty_attrs = ['name']
-    STATS_TO_MATCH = ['num_server']
-    # string_name = "Physical Infrastructure"
-    # page_name = "infrastructure"
+    STATS_TO_MATCH = ['num_server', 'num_server_with_host']
+    string_name = "Physical Infrastructure"
+    page_name = "physical_infra"
```

Comment:
**ANNOTATION:** This is used when we are setting things like the start page. In ManageIQ you can get the system to land you on a particular page on logging in. This is the "key" that is used to determine this page.